Enterprise-CMCS/cmcs-eregulations
# Eregcsc 2532 cloudfront distribution should have logging enabled

Resolves #EREGCSC-2532-cloudfront-distribution-should-have-logging-enabled

**Description-**
Setup cloudfront distribution to have logging. Add an s3 bucket to store the logs
**This pull request changes...**

- expected change 1

**Steps to manually verify this change...**
Update serverless to switch logging on and to store to a designated s3 bucket.

**steps to view and verify change**
Log into AWS
Visit the cloudfront [ERFX3UXB2RJZF](https://us-east-1.console.aws.amazon.com/cloudfront/v4/home?region=us-east-1#/distributions/ERFX3UXB2RJZF)
Click [Edit](https://us-east-1.console.aws.amazon.com/cloudfront/v4/home?region=us-east-1#/distributions/ERFX3UXB2RJZF/edit)
Notice the [cloudfront distribution](https://us-east-1.console.aws.amazon.com/cloudfront/v4/home?region=us-east-1#/distributions/ERFX3UXB2RJZF/edit) has logging switched on and s3 bucket is assigned to it to save to. 

![Screen Shot 2024-03-04 at 5 15 09 PM](https://github.com/Enterprise-CMCS/cmcs-eregulations/assets/7727136/b572d13e-2348-4268-a663-3777190f4a99)

Check the [s3 bucket](https://s3.console.aws.amazon.com/s3/home?region=us-east-1#)
Click on [eregs-dev1193-cloudfront-logs](https://s3.console.aws.amazon.com/s3/buckets/eregs-dev1193-cloudfront-logs?region=us-east-1&bucketType=general&tab=objects)
Click on the [Permissions tab](https://s3.console.aws.amazon.com/s3/buckets/eregs-dev1193-cloudfront-logs?region=us-east-1&bucketType=general&tab=permissions)
Make sure that block public access is turned on so we do not get another ATO report on that 

![Screen Shot 2024-03-04 at 5 21 59 PM](https://github.com/Enterprise-CMCS/cmcs-eregulations/assets/7727136/d7ff3111-4f1d-4600-936d-2073414de099)
